Bitmex.NET/BitmexApiProxy.cs

```diff
@@ -81,7 +83,7 @@ private Task<string> SendAndGetResponseAsync(HttpMethod method, string action, I
             return SendAndGetResponseAsync(request, content);
         }
 
-        private async Task<string> SendAndGetResponseAsync(HttpRequestMessage request, string @params = null)
+        private async Task<BitmexApiResult<string>> SendAndGetResponseAsync(HttpRequestMessage request, string @params = null)
```

Comment:
@HA07Z7XE could you please change existing integration test(s) or create a new one to with validation of these headers
I want to see` RateLimitLimit ,RateLimitRemaining ,RateLimitReset ` covered with the tests. 
you implement it in your next PR about integration test fixing

Reply:
done